main.py

```diff
@@ -37,174 +21,53 @@ class ItemsDB():
-
-    def parce_info(self, raw_date: Dict) -> Dict:
-        real_state = {}
-        for i in raw_date["items"]:
-            real_state[i["id"]] = {
-                "coords": Coords(i["coords"]["lat"], i["coords"]["lng"]),
-                "price": int(re.sub(r"\D", "", i["normalizedPrice"]))
-            }
-        return real_state
-
-
-class ViewData:
-
-    def __init__(self, storage: Storage):
-        self.storage = storage
-
-    def create_map(self, x: float, y: float):
-        city_center = [x, y]
-        data_points = []
-        print("create map")
-        for val in self.storage.return_with_filter(x + 0.02949858216,
-                                                   x - 0.02949858216,
-                                                   y + 0.05012512207,
-                                                   y - 0.05012512207).values():
-            data_points.append([val["coords"].x, val["coords"].y, val["price"]])
-
-        # Создаём базовую карту
-        m = folium.Map(location=city_center, zoom_start=12)
-
-        # Добавляем тепловую карту
-        heat_data = [[point[0], point[1], point[2]] for point in data_points]  # Координаты и интенсивность
-        HeatMap(heat_data).add_to(m)
-        m.save("heatmap.html")
-        print("map created")
 
 # storage = Storage()
 # coords_queue = RequestedCoords()
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
 
 
-
-
-def http_server(storage: Storage, coords_queue: RequestedCoords):
-
+def http_server(storage: Storage, coords_queue: RequestedCoords) -> FastAPI:
     app = FastAPI()
+
     @app.get("/get/")
-    async def get_file(x: float, y: float):
-        print("start request page")
-        coords_queue.coords.append(Coords(x,y))
+    async def get_file(x: float, y: float) -> FastAPI:
+        logger.info("Start request page")
+        coords_queue.coords.append(Coords(x, y))
         view_data = ViewData(storage)
         view_data.create_map(x, y)
-        print("page generated")
-        file_path = f"./heatmap.html"
-        print("handle request")
+        logger.info("Page generated")
+        file_path = "./heatmap.html"
+        logger.info("Handle request")
         if os.path.exists(file_path):
             return FileResponse(file_path)
         else:
             raise HTTPException(status_code=404, detail="File not found")
 
     return app
 
-async def main():
+
+async def main() -> None:
     storage = Storage()
     coords_queue = RequestedCoords()
     scrapper = Scrapper(storage, coords_queue)
     scapper_task = asyncio.create_task(scrapper.run())
 
-    # Ожидаем выполнения FastAPI-сервера
-
-    config = Config(app=http_server(storage,coords_queue), host="0.0.0.0", port=8000, log_level="info")
+    config = Config(
+        app=http_server(storage, coords_queue),
+        host="0.0.0.0",
+        port=8000,
+        log_level="info",
+    )
     server = Server(config)
     server_task = asyncio.create_task(server.serve())
```

Comment:
usually we dont start server like that. For this we utilize command line. I.e. application should not implement server startup, it is a responsibility of infrastructure's software it runs on. Of course it is not a must and not always the case, you could implement entry point (cli command) for your app, but it would look differently from your current implementation
`uvicorn main:app --host 0.0.0.0 --port 8010`